build.sh

```diff
@@ -4,13 +4,15 @@
 # the files that `bb_create` creates.
 
 
-# fail on any error
-set -e
-
+# this can fail on linux or if running on a machine without a node
+# installed
 echo "Installing npm dependencies locally..."
 cd src/basic_bot/created_files/webapp/ && npm install
 cd ../../../..
```

Comment:
I recently added this to the build.   When testing, I will upload from basic_bot to `~/basic_bot/` on the daphbot_due onboard Pi5.  Then, on the pi5, I will
```sh
cd ~/basic_bot
./install-dev.sh
```
which also calls build.sh.   After doing this (very infrequently), I can then (very frequently) just upload changes to basic_bot and restart the effected services to see changes.

Reply:
I have no intention of installing node on the onboard computer.  There is also little need so long as when I run build.sh locally, it will also build the webapp with any changes.   `./upload.sh` afterwards will upload the updated npm build files (webapp/dist) to the onboard computer.